docs/developer/architecture/alerting.rst

```diff
@@ -0,0 +1,37 @@
+Alerting Functionalities
+========================
+
+Context
+-------
+
+MetalK8s is automatically deploying Prometheus, Alert Manager and a set of
+predefined alert rules. In order to leverage Prometheus and Alert Manager
+functionalities, we need to explain, in the documentation, how to use it.
+In a later stage, those functionalities will be exposed through various
+administration and alerting UIs, but for now, we want to provide our
+administrator with enough information in order to use very basic alerting
+functionalities.
+
+Requirements
+____________
+
+As a MetalK8s administrator, I want to list or know the list of alert rules
+that are deployed on MetalK8s Prometheus cluster In order to identify on what
```

Comment:
Is the capitalized "i" intended ?
Missing comma before "In order".

Reply:
Actually I think it is good to keep capital letter for the As a, I want to, In order to formalism

Reply:
Ok, good to me, just wanted to be sure it's intended.